src/main/scala/com/etsy/sbt/Checkstyle.scala

```diff
@@ -10,56 +10,110 @@ import sbt._
 
 /**
   * An SBT plugin to run checkstyle over Java code
-  *  
+  *
   * @author Andrew Johnson <ajohnson@etsy.com>
+  * @author Alejandro Rivera <alejandro.rivera.lopez@gmail.com>
+  * @author Joseph Earl <joe@josephearl.co.uk>
   */
 object Checkstyle extends Plugin {
+
   import com.etsy.sbt.Checkstyle.CheckstyleTasks._
 
   object CheckstyleTasks {
     val checkstyle = TaskKey[Unit]("checkstyle", "Runs checkstyle")
+    val checkstyleCheck = TaskKey[Unit]("checkstyle-check", "Runs checkstyle and fails the task if issues are found")
```

Comment:
What do you think of of having a boolean setting that enables the feature of failing on issues rather than a separate task?  Alternatively `checkstyleCheckSeverityLevel` could be an `Option[String]` to enable/disable that behavior.

I'm not opposed to the current approach, just curious about your thoughts on doing it this way.